wp-content/themes/missilethreat/header.php

```diff
@@ -28,28 +28,32 @@
 
 		<?php
 		wp_body_open();
+
+		if( is_singular('post') OR is_singular('defsys') OR is_singular('missile') ) {
```

Comment:
In PHP, if statements, the OR operator is the same as it in JS `||`. But, WP actually makes this easy to look up multiple post types: `is_singular( array( 'post', 'defsys', 'missile' ) );`